src/Todo.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import './Todo.css';
+
+interface TodoItem {
+  todo: string;
+  completed: boolean;
+}
+
+const Todo = () => {
+  const [todo, setTodo] = useState<string>('');
+  const [array, setArray] = useState<TodoItem[]>([]);
```

Comment:
Instead of naming it as array you can give it more meaning full name such as todoList & the function as setTodoList